index.html

```diff
@@ -81,11 +81,11 @@ <h2>Hooks</h2>
       </section>
       <section>
         <a href="slides/07_Data_Fetching.html" target="_blank">
-          <h2>Data Fetching</h2>
+          <h2>Side Effects & Data Fetching</h2>
```

Comment:
Sorry, surely my fault for throwing everything over again :D but this needs to be adapted to just be "Data Fetching" now